lib/auth/profile.js

```diff
@@ -34,7 +34,9 @@ module.exports = (endpoint) => {
               allowedActions: meta.allowedActions
             };
           })
-          .catch(() => null);
+          .catch(err => {
+            throw err;
+          });
```

Comment:
This is a total no-op. Just remove the `catch` block completely.

Reply:
herr derp